fenix/app/src/androidTest/java/org/mozilla/fenix/ui/SitePermissionsTest.kt

```diff
@@ -286,4 +307,18 @@ class SitePermissionsTest {
             verifyPageContent("User denied geolocation prompt")
         }
     }
+
+    @SmokeTest
+    @Test
+    fun fileUploadPermissionTest() {
+        val testPage = TestAssetHelper.getHTMLControlsFormAsset(mockWebServer)
+
+        navigationToolbar {
+        }.enterURLAndEnterToBrowser(testPage.url) {
+            clickUploadButton()
+            grantPermission()
+            // Package name valid for API 11
+            assertExternalAppOpens("com.google.android.documentsui")
```

Comment:
I think you mean Android 11 / API 30. I'm not familiar with this. Is this only expected on API 30? Should this be conditionality wrapped in a `Build.VERSION_CODES` check?

Reply:
yes, I typed in API 11 by mistake, thanks! The reason is that the package name is different between android 9 and 11, at least those are the only 2 versions I've checked. I could wrap it in a version check, sure.